### PR TITLE
Move Monitoring artifacts to monitoring folder

### PR DESCRIPTION
**Pull request by anishasthana**
Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,33 +14,32 @@ RUN microdnf update -y && \
...
       org.label-schema.name="ODH deployer" \
-      org.label-schema.vcs-ref="${vcs}" \
+      org.label-schema.vcs-ref="$vcs" \
       org.label-schema.vendor="Red Hat" \
-      org.label-schema.version="${version}"
+      org.label-schema.version="$version"
 
-ENTRYPOINT [ "/deploy.sh" ]
+WORKDIR $HOME
+ENTRYPOINT [ "/home/deployer/deploy.sh" ]
```
Comment (anishasthana) [inline Dockerfile:43]: Thought here -- is it a bad idea to add $HOME to the $PATH variable?

**Review by crobby — CHANGES_REQUESTED** [review]
The entrypoint is still set to /deploy.sh  I think that will cause problems.

**Comment by anishasthana** [comment]
The entrypoint is set to ./deploy.sh . 
image

**Review by crobby — APPROVED** [review]
lgtm.  My problem was.....my problem.

**Comment by crobby** [comment]
> The entrypoint is still set to /deploy.sh I think that will cause problems.

my mistake.....this is good

**Review by crobby — APPROVED** [review]
Seems to be working well, lgtm.

**Pull request merged**